### DISCUSSION TITLE
Data modifications possible?

### DISCUSSION THREAD

**Discussion by senj** [vanna-ai/vanna#347] [Q&A]
Hello and sorry for the simple question, but I didn't find an answer anywhere.

I tried to do simple data modifications with Vanna (delete row, update value) but it doesn't work. Is it not supported or am I doing something wrong? I checked with the database and the record is still there, even when I "manually fix" it and run the sql again.

The SQL is correct and querying works just fine!

![image](https://github.com/vanna-ai/vanna/assets/5556013/3f4c8587-9fab-4a58-9864-beeaa0a76b5f)

**Comment by zainhoda** [comment]
The Vanna UI is intended for analytical queries (SELECT) where you get data back and get visualizations etc. You could shoehorn this use case where you perform arbitrary SQL but it may require some tweaks. 

You can still use the core Python package to generate and the SQL you've shown.
